charts/asserts/values.yaml

```diff
@@ -376,16 +376,12 @@ grafana:
   ## Grafana admin password configuration
   ##
   ## If left with password: "" and existingSecret: ""
-  ## a random alpha numeric password will be generated.
-  ## Upon upgrading the release, you will need to provide
-  ## the password as a value in subsequent updates.
-  ## e.g.
-  # grafana:
-  #   auth:
-  #     password: <grafana-admin-password>
-  ##
-  ## If an existingSecret is created (recommended for production)
-  ## then that will be used.
+  ## a random alpha numeric password will be generated on
```

Comment:
👍  Nice to know that we don't have to provide the password in clear text on upgrades